contrib/devtools/test-security-check.py

```
@@ -39,7 +39,7 @@ def env_flags() -> List[str]:
         flags += filter(None, os.environ.get(var, '').split(' '))
     return flags
 
-def call_security_check(cc, source, executable, options):
+def call_security_check(cc: str, source: str, executable: str, options) -> tuple:
```

Ruff (0.12.2)

Missing type annotation for function argument `options`

(ANN001)